src/youtube-player/youtube-player.ts

```diff
@@ -153,8 +165,7 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
   @Output() playbackRateChange = new EventEmitter<YT.OnPlaybackRateChangeEvent>();
 
   /** The element that will be replaced by the iframe. */
-  @ViewChild('youtubeContainer')
-  youtubeContainer: ElementRef<HTMLElement>;
+  @ViewChild('youtubeContainer', {static: true}) youtubeContainer: ElementRef<HTMLElement>;
```

Comment:
From what I can tell we don't need the query to be `static`. Am I missing something?